src/platform/selfsign_capi.c

```diff
@@ -1031,13 +1031,25 @@ FindOrCreateCertificate(
     return (void*) Cert;
 }
 
+_IRQL_requires_max_(PASSIVE_LEVEL)
+const char*
+CxPlatGetSelfSignedCertCaCertificateFileName(
+    _In_ BOOLEAN ClientCertificate
+    )
+{
```

Comment:
you'll also need a `UNREFERENCED_PARAMETER(ClientCertificate)` here, otherwise the compiler will yell.

Reply:
Right, missed that one.